html/index.html

```diff
@@ -45,69 +45,69 @@ <h2 class="text-center">Image card container</h1>
 <!-- Set to http://localhost:8080/gxa/ or http://localhost:8080/gxa/sc/ -- Remember the trailing slash! -->
 <script>
   homepageCardsDemo.renderExtendableCard({
-    iconType: `image`,
-    iconSrc: `https://upload.wikimedia.org/wikipedia/en/6/6b/A_Rickle_in_Time.jpg`,
+    iconType: 'image',
```

Comment:
Oh? I thought it was meant to be backticks everywhere?

Reply:
Most moder browsers can interpret them correctly, but I wouldn’t bet on that. Notice the semicolons, too.